lib/src/utils.dart

```diff
@@ -530,6 +533,12 @@ Map<String, Object> getExtensions(
   }
 
   context.path.add(EXTENSIONS);
+
+  if (warnOnMultipleExtensions && extensionMaps.length > 1) {
+    context.addIssue(SemanticError.multipleExtensions,
+        args: [null, extensionMaps.keys]);
+  }
```

Comment:
Not sure I understand this one ... why is having multiple extensions reported as an issue? Your test case makes sense — multiple _material_ extensions may interact in undefined ways — but reporting an issue when KHR_materials_unlit is used with KHR_draco_mesh_compression makes less sense.

Reply:
This warning is effective only when several extensions are defined for the same object.
For now, it's enabled only for `material` objects:
https://github.com/KhronosGroup/glTF-Validator/blob/d376c03d0764a27401d8867e640b9f91908f5686/lib/src/base/material.dart#L89-L90

Reply:
Sounds good.

Reply:
Done. Please review.